multi-agent/run/scripts/migrate_builtin_system.py

```
@@ -0,0 +1,316 @@
+import pymongo
+
+from global_utils.utils.crypto import FieldCipher, FERNET_PREFIX
+from mas.catalog.element_registry import ElementRegistry
+from mas.core.enums import ResourceCategory
+from mas.resources.field_encryption import ResourceFieldEncryption
+
+logging.basicConfig(level=logging.INFO, format="%(levelname)s: %(message)s")
+logger = logging.getLogger(__name__)
+
+
+def run_migration(db_name: str, mongodb_ip: str, mongodb_port: str, dry_run: bool, reverse: bool = False, encryption_key: str = "") -> None:
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)